### PR TITLE
UHM-4934: Create "circular workflow" for adding Lab Orders from the L…

### PR DESCRIPTION
**Pull request by mogoodrich**
…ab Workflow module

### CODE REVIEW

**Review by cioan — APPROVED** [review]
Looks good @mogoodrich !

File: app/js/components/LabOrdersList/LabOrdersList.jsx [hunk 1]
```diff
@@ -86,26 +86,32 @@ export class LabOrdersList extends PureComponent {
...
 
-    // if a specific order uuid has been specified, redirect to the display
-    // page for that order after the orders have been fetched
-    if (fetched && !prevProps.fetched && match.params.orderUuid && orders) {
+    // if a specific order uuid (or set of order uuids) has been specified, redirect to the display
+    // page for first order after the orders have been fetched; set a "afterSubmitLink" with the
+    // remaining order ids so we cycle through them; this provides a way for our Order Entry app to
+    // redirect to the entry pages for a set of orders after placing the order
+    if (fetched && !prevProps.fetched && match.params.orderUuids && orders) {
+      const [orderUuid, ...otherOrderUuids] = match.params.orderUuids.split(",");
```
Comment (mseaton) [inline app/js/components/LabOrdersList/LabOrdersList.jsx:94]: Interesting syntax - this works?

**Review by mseaton — APPROVED** [review]
All seems reasonable.  Code looks fine.

File: app/js/components/LabOrdersList/LabOrdersList.jsx (hunk 1 above)
Comment (mogoodrich) [reply app/js/components/LabOrdersList/LabOrdersList.jsx:94]: Yep, the spread & rest operators and destructing are the best practice way (I think?) of doing this kind of stuff in ESM6

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

**Pull request merged**